src/Samples/DIPS.Xamarin.UI.Samples/App.xaml.cs

```diff
@@ -12,6 +12,7 @@ public App()
             //This has to be called once per application in order to use Custom Namespace in XAML : https://github.com/DIPSAS/DIPS.Xamarin.UI/issues/1
             DIPS.Xamarin.UI.Library.PreviewFeatures.EnableFeature("SkeletonView");
             DIPS.Xamarin.UI.Library.Initialize();
+            DIPS.Xamarin.UI.Controls.Toast.Toast.Current.Initialize();

```

Comment:
In my test app I actually didn't do this and the toast still worked. Maybe you could have another look if it is required?

Reply:
This is needed for pages that does some content changes, else it won't be needed.

Reply:
This can be removed as the problem was with our internal apps XAML file. The toast actually highlighted a problem we had with the app that we hadn't noticed before. 👍

Reply:
Adding this in Library as an internal method, to set the Toast Content on page loading to make sure no UI states are changed on displaying the Toast (ex: scrolled position)

Reply:
That's fine 👍